pkg/cli/mirror/operator.go

```diff
@@ -705,3 +717,29 @@ func validate(dc *declcfg.DeclarativeConfig) error {
 	}
 	return nil
 }
+
+// checkError will check validation errors
+// from operator registry and return a modified
+// error messages for mirror usage
+// FIXME(jpower432): Checking against the errors string could be an issue since
+// this is depending on the strings returned from `opm validate`. It would be better to propose that
+// the validation error are exposed here https://github.com/operator-framework/operator-registry/blob/master/alpha/model/error.go
+// and adds structure errors that return package and channels information.
+func (o *OperatorOptions) checkValidationErr(err error) error {
+	if err == nil {
+		return nil
+	}
+
+	fmt.Fprintln(o.ErrOut, "\nThe rendered catalog is invalid.")
+	// handle known error causes
+	var validationMsg string
+	switch {
+	case strings.Contains(err.Error(), "channel must contain at least one bundle"):
```

Comment:
Hopefully one day we won't need to do this ugly string check for validation in FBC.